dolo/compiler/model_numeric.py

```diff
@@ -179,6 +179,14 @@ def _repr_html_(self):
             equations = {"dynare": self.symbolic.equations}
         else:
             equations = self.symbolic.equations
+            # Create definitions equations and append to equations dictionary
+            definitions = self.symbolic.definitions
+            tmp = []
+            for deftype in definitions:
+                tmp.append(deftype + '=' + definitions[deftype])
+            definitions = {'definitions': tmp}
+            equations.update(definitions)
```

Comment:
Sure. If it's quick, feel free to make the change yourself. If not I can have a go next week.